roles/confluent.common/tasks/failure_handling.yml

```diff
@@ -15,13 +15,46 @@
     recurse: false
   register: find_output
 
+- name: Remove directory
+  file:
+    path: "/tmp/error_files/{{inventory_hostname}}/"
+    state: absent
+
+- name: Recreate directory
+  file:
+    path: "/tmp/error_files/{{inventory_hostname}}/"
+    state: directory
+    mode: 0750
+    owner: "{{user}}"
+    group: "{{group}}"
+
+- name: Copy Config and Log Files to tmp dir
+  copy:
+    dest: "/tmp/error_files/{{inventory_hostname}}/"
+    src: "{{ item }}"
+    remote_src: true
+    mode: 0750
+    owner: "{{user}}"
+    group: "{{group}}"
+  loop: "{{ find_output.files | map(attribute='path') | list + [config_file] }}"
+
+- name: Archive log files
```

Comment:
This is awesome, the only thing i can think of is removing the stuff in /tmp after you fetch. WDYT?

Reply:
Good point. Also, the directory name can be renamed. Will give a try.